bash/atomic

```diff
@@ -651,12 +651,109 @@ _atomic_migrate_import() {
 
 }
 
+_atomic_cluster_no_opts() {
```

Comment:
@rhatdan I can, but are you sure?  Nothing else under `Atomic/` appears to be serving bash completion.  Just wanted to double check first.